jetty-ee9/jetty-ee9-demos/jetty-ee9-demo-proxy-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -13,7 +13,7 @@
     <servlet-class>org.eclipse.jetty.ee9.proxy.ProxyServlet$Transparent</servlet-class>
     <init-param>
       <param-name>proxyTo</param-name>
-      <param-value>https://eclipse.dev/jetty/javadoc/jetty-12/index.html?overview-summary.html</param-value>
+      <param-value>https://jetty.org/javadoc/jetty-12/index.html?overview-summary.html</param-value>
```

Comment:
Wrong URL

Reply:
This one has both the hostname fault, and the query section fault.

Reply:
Checked for other ones in `jetty-12.0.x`, but just found one that is a valid test case.

``` shell
[12.0.x]$ git grep "jetty.org/.*?"
jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/FileIDTest.java:            Arguments.of(URI.create("https://jetty.org/docs.html?query=val#anchor"), "docs.html")
```

Reply:
fixed